test/test_commands_serve.rb

```diff
@@ -79,16 +85,53 @@ def custom_opts(what)
         custom_options = {
           "config"  => %w(_config.yml _development.yml),
           "serving" => true,
-          "watch"   => false # for not having guard output when running the tests
+          "watch"   => false, # for not having guard output when running the tests
+          "url"     => "http://localhost:4000"
```

Comment:
This test checks which options are passed to `Serve.process` and the URL is now also included in that options hash.